articles/how-to-insert-3d-objects-into-a-webpage-using-html-and-css/index.md

```diff
@@ -0,0 +1,320 @@
+Some popular formats available include *FBX*, *STL*, *OBJ*, *glTF*, and *DAE* among others.
+In our case, we will use the *glTF model*.
+
+> **glTF**, *Graphics Language Transmission Format*, is a 3D model format designed for efficient transmission and loading of 3D models by applications.
+
+After visiting the above sites or any other, search for any preferable model and download it, in this case, we are going to search, download and use a Virtual Reality Headset (VR Headset) model which is in glTF format. 
+
+![VR Headsets](/engineering-education/How-to-insert-3D-objects-into-a-webpage-using-HTML-and-CSS/poly-VR-Headset.png)
+
+## 3D Model Attributes
+These are the characteristics of the viewed 3D Object on the website set by the frontend developers as preferred. Some attributes one can set to 3D models include:
+- Auto-Rotate
+- Alternative text(alt)
+- Model-Visibility
+- Augmented Reality
+- Touch-action
+- Camera-controls
+- Source(src)
+- ios-src and many more...
+
+
+**Let's now get into the tutorial:**
+First things first, let's set up our Visual Studio Code IDE. In the extensions tab, search for the "HTML Boilerplate" and "Live Server" extensions and install them. Create a folder and name it "3D Objects". Open the folder with Visual Studio Code and in it create other two folders namely "assets" and "css". Create an "index.html" file in the main directory and "style.css" in the css folder. 
+
+## Create a webpage Strucure
+We are going to create a simple HTML structure. We can simplify the process using boilerplate generators which may come pre-installed by default in Visual Studio Code or as downloadable extensions by typing in the index file, "html:5" or "html5-boilerplate" then key in the Enter key. Since we don't need additional elements in the boilerplate generated, we will delete the extra code and link it to our CSS file. Let's also link it with some online scripts which will load some Font Awesome fonts and icons. The outcome will be as shown below:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>3D Object</title>
+    <script src='https://kit.fontawesome.com/a076d05399.js'></script>
+    <link rel="stylesheet" href="css/style.css">
+    
+</head>
+<body>
+    
+</body>
+</html>
+```
+Start the live server by either right-clicking on the "index.html" file in the explorer and selecting "Open with Live server" or on the status bar clicking on "Go live" to start the server. It will automatically open a preview of the webpage in the default browser which will be blank because we haven't added any content yet. I recommend you use chrome since it supports most 3D objects. You can change the default browser in the settings. If it doesn't automatically open the webpage preview in the browser, check the port number in which the server uses at the Visual Studio Code status bar then add access it in the browser by typing in the URL:
+
+~~~~
+http://127.0.0.1:5501/
+~~~~
```

Comment:
You could just use something like for highlighting `http://127.0.0.1:5501/` (Most preferred) or if you want something like a code snippet. It can be done this way

```
http://127.0.0.1:5501/
```

Reply:
Got it